src/main/java/org/rulez/demokracia/pdengine/dataobjects/CastVote.java

```diff
@@ -20,4 +21,9 @@ public CastVote(final String proxyId, final List<RankedChoice> preferences) {
 	public List<RankedChoice> getPreferences(){
 		return this.preferences;
 	}
+
+	public List<String> getAssurances() {
```

Comment:
Well, we have a lesson here to be learned. Thank you for making me thinking about it.
But first tell me why is it not a zombie?

Reply:
Please define zombie.
This method is the scope of the #242. I didn't want to pick it up, because @valentinbujdoso has been assigned on it. I think the merge conflict will make it obvious, which implementation would be kept.

Reply:
mutant: modified production code
killed mutant: a mutant which is detected by failing unit tests
zombie: an unkilled mutant
Well, it was not covered by tests so obviously, that mutation analysis did not even consider to check it.
I will now review your code assuming that my proposal about the case will stand. So the validity of the review is tied to the status of that proposal.